bindings/python/math/expose-rpy.cpp

```diff
@@ -0,0 +1,59 @@
+//
+// Copyright (c) 2019 CNRS INRIA
+//
+
+#include "pinocchio/bindings/python/fwd.hpp"
+#include <boost/python.hpp>
+//#include "pinocchio/bindings/python/utils/namespace.hpp"
+#include "pinocchio/math/rpy.hpp"
+
+namespace pinocchio
+{
+  namespace python
+  {
+    namespace bp = boost::python;
+
+    Eigen::Matrix3d rpyToMatrix_proxy(const Eigen::Vector3d & rpy)
+    {
+      return pinocchio::rpy::rpyToMatrix(rpy);
+    }
+
+    Eigen::Vector3d matrixToRpy_proxy(const Eigen::Matrix3d & R)
+    {
+      return pinocchio::rpy::matrixToRpy(R);
+    }
+    
+    void exposeRpy()
+    {
+      using namespace Eigen;
+      using namespace pinocchio::rpy;
+
+      {
+        // using the rpy scope
+        //bp::scope current_scope = getOrCreatePythonNamespace("rpy");
+        
+        bp::def("_rpyToMatrix",
```

Comment:
Why did you put some `_` as prefix?

Reply:
I did not want them to appear in the main `pinocchio` module. I manually import them in the `rpy` submodule

Reply:
It changes nothing to directly import them,no?

Reply:
The current solution keeps them hidden in the main Pinocchio module but displays them in the `pinocchio.rpy` submodule instead.
I think I could do `from pin import _rpyToMatrix as rpyToMatrix`, it would be the same.
I had unexpected errors creating the submodule from the C++ bindings.
Also, I wanted to keep the Python implementation for `npToTTuple`, `npToTuple` and `rotate` and I did not how the C++-created module would interact with the Python module.
This is the quickest solution I found. I may look for something else, but I think it is a waste of time, there are more urgent things

Reply:
> I had unexpected errors creating the submodule from the C++ bindings.

Not clear. What do you mean here?

Reply:
What I mean is: `bp::def("rpyToMatrix")` would expose the function in the main pinocchio namespace. I did not want that.
So I did `bp::def("_rpyToMatrix")` to hide it. Then I imported it as `rpyToMatrix` inside `pinocchio.rpy`.

Also: I had trouble with `getOrCreatePythonNamespace` (duplicate symbols).
Even if I _did_ manage to use `getOrCreatePythonNamespace("rpy")`, it would have conflicted with the existing Python submodule `pinocchio.rpy`, which I wanted to keep in order not to bother with the C++ implementation of `npToTTuple`, `npToTuple` and `rotate`.
Is it clearer now?

Reply:
OK. I've now understood.